README.md

```diff
@@ -0,0 +1,91 @@
+# ``AzureAppConfiguration``
+
+The AzureAppConfiguration package allows for simple construction of
+a `URLRequest` for Azure's App Configuration Service, and decoding
+the response from this service into a dictionary.
+
+A discussion of what Azure App Configuration can be found [here](https://docs.microsoft.com/en-us/azure/azure-app-configuration/overview).
+
+## Overview
+
+The AzureAppConfiguration structure contains a pair of static helper
+functions; no instantiation of an AzureAppConfiguration structure is
+needed or helpful.
+
+Typically, you'd use this function one time at the start of your application,
+or whenever you want to check if the configuration has changed post-launch.
+
+A very simple example can be found in the unit tests, and is also below:
+
+```
```

Comment:
I'm not sure how much of a difference it makes formatting wise, but we can technically denote this code block as being Swift code.
```swift
```swift
```

Reply:
Thanks, done!